client/src/components/FlightBarogramm.vue

```diff
@@ -1,73 +1,37 @@
 <template>
```

Comment:
Wenn du hier eh schon am File dran bist.

Die Elevation sollte in den Hintergrund rücken. Aktuell kann es vorkommen, dass die Höhe teilw. hinter der Elevation liegt.

Reply:
done